setup.py

```diff
@@ -435,6 +464,26 @@ def finalize_options(self):
     libs.extend(qnn_deps)
     if nightly_build:
         libs.extend(["onnxruntime_pywrapper.dll"])
+    migraphx_deps = [
```

Comment:
Similar to Scott's question: https://github.com/microsoft/onnxruntime/pull/25583/files#r2250309763

Should the DLLs be distributed separately ? As we already have auto EP.

Reply:
Where are you going to publish this python package? And, what's the package name? (We might have a problem here)

Reply:
I fixed the name of the Python wheel to `onnxruntime_migraphx-*.whl`